src/eXpansion/Bundle/VoteManager/Plugins/VoteManager.php

```diff
@@ -151,5 +152,30 @@ public function onPostLoop()
     {
         // Nothing
     }
+
+    /**
+     * Callback sent when the "onPodiumStart" section start.
+     *
+     * @param int $time Server time when the callback was sent
+     * @return void
+     */
+    public function onPodiumStart($time)
+    {
+        //nothing
+    }
+
+    /**
+     * Callback sent when the "onPodiumEnd" section end.
+     *
+     * @param int $time Server time when the callback was sent
+     *
+     * @return void
+     */
+    public function onPodiumEnd($time)
```

Comment:
I am no ok with this it depends upon the vote. Why cancel a ban vote for exemple? Need to be on the plugin that handles that vote. Which is already the case if that doesen't work then needs to be a an issue